### ISSUE TITLE
octoprint with 2.0.1

### ISSUE THREAD

**Issue by Ab2dx** [Jyers/Marlin#1958]
hi 
I'm new to the world of 3D printing and I downloaded your update E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin

On my ender 3v2 printer with a 4.2.2 board it worked without problems until I connected my raspberry pi 3 b

**With 5V stripped USB cable**

And I installed the octoprint system, the latest version and I thought everything was fine, and from here the problems started

In the event that you connect, an error screen appears in the printer bearing the code M112

IF YOU PRINTER AND OCTOPRINT ? Connected

I cannot print at all because of errors that occur during printing and frequent stops with unrelated problems. It is also not possible to cancel printing completely from the printer itself or from the external system, even despite the separation of OCTO, the printer remains in the printing cycle without exit without stopping, Slow to respond, freezes for a moment even sometimes it prints even if I succeeded in canceling, and if I restart the printer again and connect, the error code M112 appears!!

Your update is good and I don't want to give it up, but I'm completely unbalanced with OCTOPRINT

So what is the solution

**Comment by tititopher68-dev** [comment]
@Ab2dx 
Look at: https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings and select 'Ignore warning'

**Comment by Ab2dx** [comment]
OK cool , but how I do those settings there is no instruction for that ???! 
 
Do I need to flash a printer again with those commands ??

**Comment by Ab2dx** [comment]
Yeah my bad I just realise I must copy those command in **_OctoPrint Settings_** on **GCODE Scripts**

thank you for helping and I will try it right now

**Comment by Ab2dx** [comment]
I wrote the settings, and in fact all the problems were solved except, a problem if I canceled the printing process from the printer itself, here there is a repetition and a cycle of errors and not canceling the printing process completely and this thing can only be addressed by restarting the printer, this does not happen often but it almost happened  More than once, everything seems normal

**Comment by tititopher68-dev2** [comment]
@Ab2dx 
A fact of life. When printing via USB, the printer gets fed gcode commands one at a time. The printer has no idea of the full gcode job, it just sees one command and then the next. This is different from printing from SD on the printer, because then the printer knows that it is executing gcode commands from a longer file.

When canceling an sd print on the printer, the printer knows to stop processing the file. The problem you are faacing is that you tell the printer to stop printing, but the printer does not tell OctoPrint to stop sending gcode lines. OctoPrint is unaware that you would like it to stop sending gcode.

The question is : do you use the configuration files on my sources ? If not, check in your configuration files if HOST_ACTIONS_COMMAND is enable. Without this parameter, you can't cancel a host print.

**Comment by Ab2dx** [comment]
Now I understand the problem and I know that it does not happen unless you print it with { octoprint } connected. As for your question, the answer is: No, and I do not know how to open or modify the configuration files. Can you explain that, thank you

**Comment by tititopher68-dev2** [comment]
@Ab2dx 
Just give me your motherboard version, CPU Type, BLTouch or not, Bilinear mode, 3x3 , 5x5 ? I send you the firmware complied.

**Comment by Ab2dx** [comment]
That would be great

Motherboard version :
4.2.2 
CPU type :
 I didn't know actually 
I am in the work right now far away from the house,i well looking directly on motherboard ,
 And  I will provide you the CPU model , type

 

Software : 
E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin

BLTouch or not :
Right now I don’t have the CRTouch ,
{{ I already buy it , so I will have it in a few days }}



So that meaning 
If you modify the firmware , I need 2 copy

1  for BLTouch 5x5 
1  for Manuel Mish 5 x 5 




Thank you very much for helping me

**Comment by Ab2dx** [comment]
Quick update # 


I am in the work right now far away from the house, and I realize you can check  your Cpu type by looking directly on  motherboard , 

So when I get home I will provide you the CPU model , type

Thank you again

**Comment by tititopher68-dev** [comment]
@Ab2dx 
[firmwares.zip](https://github.com/Jyers/Marlin/files/8699526/firmwares.zip)

I compiled with type CPU RCT6....100% OK with RET6 !

**Comment by tititopher68-dev** [comment]
@Ab2dx 
Don't forget to add Set and Wait plugin to easy cancel a print ! it's an issue with Octoprint...

**Comment by Ab2dx** [comment]
hi again I confirm my CPU is RCT 6

and I just download the plugging as you said

and I installed BL touch firmware 
and when I start the  auto-leveling  on the right corner or in the center I realize there is something wrong
the nozzle far away from the bed
I mean the nozzle is outside the bed literally

then  I checked the info system information
I realize that 
it 230x230x250

what I really know is this is not a v2 bed size
so how do I change those numbers

**Comment by tititopher68-dev** [comment]
you said 'and when I start the auto-leveling on the right corner or in the center I realize there is something wrong
the nozzle far away from the bed
I mean the nozzle is outside the bed literally'
It's a normal behaviour with a probe.

'I realize that
it 230x230x250' ->
It's the limit size for an Ender3V2 ! We know what we are doing, it has always been like that!...I think you have a lot to learn...And before to incriminate a firmware, just ask yourself the question if the problem does not come from your practice and lack of knowledge. Instead, ask open-ended questions, 'I'm doing this, is this right?' or 'I don't understand this or that, can you help me or explain it to me...' because there, you are going to face big difficulties... which are not in reality ;-)

**Comment by Ab2dx** [comment]
Oh I think you understand me wrong
I'm new in this world literally I don't know what's the right and what's the wrong
and from the beginning, I'm really thankful for what you doing really
you give me your time you try to give me the best experience and I just noted how much you did and this software


and you are right I have a lot to learn
so please  please  do not be offended by my questions and like you said my questions come from a lack of acknowledge



so if you feel that I'm really sorry and thank you for what you doing bro

**Comment by tititopher68-dev2** [comment]
@Ab2dx 
This is not at all reproaches but rather advice so that those who can help, do it in the most appropriate way....I have learned and continue to learn from the experience of others and  to their advice.  And in 3D printing, we all learn, all the time.

**Comment by Ab2dx** [comment]
In the end, I would like to thank you, because the problem has been completely eliminated, and I am very impressed with this copy that you gave me because it contains exceptional details and capabilities.


Really amazing and thank you for what you did


I would also like to clarify a certain point: I think I misunderstood or made an inappropriate statement, although, as I said earlier, I do not know what is right or wrong in this field. With these circumstances, I think that the message was delivered wrongly, and the English language is not my mother tongue.


In general, I apologize once again and thank you for your patience, cooperation and time



I wish you a happy day
Your brother: Abad
